behave_django/management/commands/behave.py

```diff
@@ -21,45 +21,63 @@ def add_command_arguments(parser):
         help="Don't create a test database. USE AT YOUR OWN RISK!",
     )
     parser.add_argument(
-        '--keepdb',
+        '-k', '--keepdb',
         action='store_true',
         default=False,
         help="Preserves the test DB between runs.",
     )
 
 
-def add_behave_arguments(parser):
+def add_behave_arguments(parser):  # noqa
     """
     Additional command line arguments extracted directly from behave
     """
 
+    # Option strings that conflict with Django
     conflicts = [
         '--no-color',
-        '--version'
+        '--version',
+        '-c',
+        '-k',
+        '-v'
```

Comment:
Sorry to be picky, can we put a `,` (comma) at the end of each list item if it's on a single line? This way we never have changes in existing lines when elements are added.